### DISCUSSION TITLE
《Hit AI & Blockchain》立法院長游錫堃：區塊鏈技術要在台灣加速應用，需有良好的基礎建設配套措施！

### DISCUSSION THREAD

**Discussion by 20010906** [1101-GS4719-C/Python-Community-GS4719-C#256] [科技新知]
希望未來台灣產業能結合區塊鏈技術成功升級轉型
[台灣的區塊鏈技術](https://tw-news-yahoo-com.cdn.ampproject.org/v/s/tw.news.yahoo.com/amphtml/hit-ai-blockchain-%E7%AB%8B%E6%B3%95%E9%99%A2%E9%95%B7%E6%B8%B8%E9%8C%AB%E5%A0%83-%E5%8D%80%E5%A1%8A%E9%8F%88%E6%8A%80%E8%A1%93%E8%A6%81%E5%9C%A8%E5%8F%B0%E7%81%A3%E5%8A%A0%E9%80%9F%E6%87%89%E7%94%A8-033000612.html?amp_js_v=a6&amp_gsa=1&usqp=mq331AQKKAFQArABIIACAw%3D%3D#aoh=16413604561352&referrer=https%3A%2F%2Fwww.google.com&amp_tf=%E4%BE%86%E6%BA%90%EF%BC%9A%251%24s&ampshare=https%3A%2F%2Ftw.news.yahoo.com%2Fhit-ai-blockchain-%25E7%25AB%258B%25E6%25B3%2595%25E9%2599%25A2%25E9%2595%25B7%25E6%25B8%25B8%25E9%258C%25AB%25E5%25A0%2583-%25E5%258D%2580%25E5%25A1%258A%25E9%258F%2588%25E6%258A%2580%25E8%25A1%2593%25E8%25A6%2581%25E5%259C%25A8%25E5%258F%25B0%25E7%2581%25A3%25E5%258A%25A0%25E9%2580%259F%25E6%2587%2589%25E7%2594%25A8-033000612.html)

**Comment by zixunn** [comment]
近年來台灣科技業發展越來越進步，期望台灣能更好。

**Comment by chun1203** [comment]
台灣加油！

**Comment by syt-110** [comment]
嘎由
